stix_shifter_modules/azure_sentinel/stix_translation/json/to_stix_map.json

```diff
@@ -30,19 +26,32 @@
       "key": "x-msazure-sentinel-alert.assignedTo",
       "object": "alert"
     },
-    "category": {
-      "key": "x-msazure-sentinel-alert.category",
-      "object": "alert"
-    },
-    "closedDateTime": {
-      "key": "x-msazure-sentinel-alert.closedDateTime",
-      "object": "alert"
+    "category": [
+      {
+      "key": "x-ibm-finding.finding_type",
+      "object": "finding"
+      },
+      {
+      "key": "x-oca-event.category",
+      "object": "event"
+      }
+    ],
```

Comment:
Looks redundant to map `category` to both x-ibm-finding and x-oca-event object

Reply:
Yeah, it should be removed from the finding object and left with the event object.

Reply:
fixed